ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1672,39 +1677,42 @@ export const WorkspaceEdit = fp.flow(
                           />
                         </div>
                       </FlexColumn>
-                      <FlexColumn>
-                        <Button
-                          disabled={this.state.fetchBillingAccountError}
-                          type='primary'
-                          style={{
-                            marginLeft: '20px',
-                            fontWeight: 400,
-                            height: '38px',
-                            width: '220px',
-                          }}
-                          onClick={() =>
-                            this.setState({
-                              showCreateBillingAccountModal: true,
-                            })
-                          }
-                        >
-                          CREATE BILLING ACCOUNT
-                        </Button>
+                    </TooltipTrigger>
+                    <FlexColumn>
+                      <Button
```

Comment:
I tried to have the TooltipTrigger cover the button as well, but I was not successful